app/controllers/users_controller.rb

```diff
@@ -55,14 +55,14 @@ def create
     user[:verified] = false
 
     if !user[:password] || user[:password].strip.empty?
-      user[:password] = 'arike'
+      user[:password] = "arike"
```

Comment:
Oh because of password_digest🤦‍♂️ but I think there is custom active record validation https://guides.rubyonrails.org/active_record_validations.html#performing-custom-validations